TELCODOCS-2045 - Telco core RDS 4.18 docs (#88622)
openshift/openshift-docs: modules/telco-core-load-balancer.adoc

// Module included in the following assemblies:
//
// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc

[id="telco-core-load-balancer_{context}"]
= Load Balancer

New in this release::
// https://issues.redhat.com/browse/CNF-14150

> * FRR-K8s is now available under the Cluster Network Operator.

Author: Not an error
https://github.com/redhat-documentation/vale-at-red-hat/pull/918

// Module included in the following assemblies:
//
// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc

:_mod-docs-content-type: REFERENCE
[id="telco-core-load-balancer_{context}"]

> = Load Balancer

Reviewer: Why capitalize the B here?

New in this release::
// https://issues.redhat.com/browse/CNF-14150
* FRR-K8s is now available under the Cluster Network Operator.
+
[IMPORTANT]
====
If you have custom `FRRConfiguration` CRs in the `metallb-system` namespace, you must move them under the `openshift-network-operator` namespace.
====

Description::
MetalLB is a load-balancer implementation for bare metal Kubernetes clusters that uses standard routing protocols.
It enables a Kubernetes service to get an external IP address which is also added to the host network for the cluster.
The MetalLB Operator deploys and manages the lifecycle of a MetalLB instance in a cluster.

> Some use cases might require features not available in MetalLB, for example stateful load balancing.

Reviewer 2: ```suggestion
Some use cases might require features not available in MetalLB, such as stateful load balancing.
```

Where necessary, you can use an external third party load balancer.
Selection and configuration of an external load balancer is outside the scope of this specification.

> When an external third party load balancer is used, the integration effort must include enough analysis to ensure all performance and resource utilization requirements are met.

Reviewer 3: third-party
